antarest/study/web/raw_studies_blueprint.py

```diff
@@ -203,6 +216,104 @@ def get_study(
         ).encode("utf-8")
         return Response(content=json_response, media_type="application/json")
 
+    @bp.get(
+        "/studies/{uuid}/areas/aggregate",
+        tags=[APITag.study_raw_data],
+        summary="Retrieve Aggregated Areas Raw Data from Study Output",
+    )
+    def aggregate_areas_raw_data(
+        uuid: str,
+        output_name: str,
+        query_file: AreasQueryFile,
+        frequency: MatrixFrequency,
+        mc_years: str = "",
+        areas_names: str = "",
+        columns_names: str = "",
+        current_user: JWTUser = Depends(auth.get_current_user),
+    ) -> t.Dict[str, t.Any]:
```

Comment:
Could we replace `Dict`  by `Mapping`?

Reply:
@laurent-laporte-pro   

The JSON format is defined as t.Dict[str, t.Any]. 
I think we should keep it.